cadl-extension/src/code-model-builder.ts

```diff
@@ -1763,6 +1783,14 @@ function operationContainsJsonMergePatch(op: HttpOperation): boolean {
   return false;
 }
 
+function pascalCase(name: string): string {
+  if (name.length > 0) {
```

Comment:
Maybe a null check?

Reply:
should be ok; if allow null, the signature would be `string | undefined` or `string | null`